examples/systemd/rest-server.service

```diff
@@ -10,13 +10,35 @@ Group=www-data
 ExecStart=/usr/local/bin/rest-server --path /path/to/backups
 Restart=always
 RestartSec=5
+# Makes created files group-readable, but inaccessible by others
+UMask=027
 
-# Optional security enhancements
+# Recommended security enhancements using features present in systemd version 247
```

Comment:
I'd like to add a remark that users can just remove options not yet available in their systemd version. After all 247 is relatively new.